inflation-inject-processor/src/test/java/com/squareup/inject/inflation/processor/InflationInjectProcessorTest.kt

```diff
@@ -149,7 +149,6 @@ class InflationInjectProcessorTest {
         .generatesSources(expectedFactory)
   }
 
-  @Ignore("Override has parameter ordering based on constructor and not factory")
```

Comment:
This probably works for regular factories now as well. Will add a test in a follow-up commit.